texttospeech/cloud-client/src/main/java/com/example/texttospeech/SynthesizeText.java

```diff
@@ -134,17 +180,29 @@ public static void main(String... args) throws Exception {
         ArgumentParsers.newFor("SynthesizeText")
             .build()
             .defaultHelp(true)
-            .description("Synthesize a text or ssml.");
+            .description("Synthesize a text, text with audio effect profiles, or ssml.");
 
     MutuallyExclusiveGroup group = parser.addMutuallyExclusiveGroup().required(true);
-    group.addArgument("--text").help("The text file from which to synthesize speech.");
+    group
+        .addArgument("--text")
+        .help("The text file from which to synthesize speech.")
+        .nargs("+")
```

Comment:
This should probably be a different argument? `.addArgument("--effect-profile").required(false)` or something?

Reply:
I think I'll just drop the CLI entirely like we did for Dialogflow

Reply:
👍